scripts/js/lib/api/saveImages.ts

```diff
@@ -18,6 +18,23 @@ import { Pkg } from "./Pkg.js";
 import { Image } from "./HtmlToMdResult.js";
 import { pathExists, rmFilesInFolder } from "../fs.js";
 
+function skipReleaseNote(imgFileName: string, pkg: Pkg): boolean {
+  const isReleaseNote =
+    imgFileName.includes("release_notes") ||
+    imgFileName.includes("release-notes");
+  if (!isReleaseNote) return false;
+
+  if (pkg.hasSeparateReleaseNotes()) {
+    // If the pkg has dedicated release notes per release, we should copy its images
+    // unless it's the dev version. We don't (yet) support release notes for dev versions:
+    // https://github.com/Qiskit/documentation/issues/1296
+    return pkg.isDev();
+  } else {
+    // Else, if the pkg has only a single release note file, only 'latest' should copy images.
+    return pkg.isLatest();
```

Comment:
Shouldn't this return true when pkg is not latest?
```suggestion
    return !pkg.isLatest();
```

Reply:
Oof, good catch!